tests/integration_tests/test_integration.py

```diff
@@ -261,15 +263,20 @@ def test_3_9_variant_qc(self, mock_args):
         except Exception as e:
             self.fail(f"Step 3.9 failed with an exception: {e}")
 
-    def test_4_1a_genotype_qc(self):
-        qc_step_4_1a = importlib.import_module("4-genotype_qc.1a-apply_range_of_hard_filters")
+    ### === Genotype QC === ###
+    @patch(
+        "argparse.ArgumentParser.parse_args",
+        return_value=argparse.Namespace(prepare=True, evaluate=True, all=False),
+    )
+    def test_4_1_genotype_qc(self, mock_args):
+        qc_step_4_1 = importlib.import_module("4-genotype_qc.1-compare_hard_filter_combinations")
         try:
-            qc_step_4_1a.main()
+            qc_step_4_1.main()
```

Comment:
I got the following error when running the test for this step:
E           Error summary: HailException: arguments refer to no files: Vector(file:///lustre/scratch126/teams/hgi/eh19/wes-qc/tests/test_data/resources/HG001_GRCh38_benchmark.interval.illumina.vcf.gz).

Is the file supposed to be downloaded from s3 automatically?

Reply:
All required files are added to the S3 bucket and the list of downloaded files.